### PR TITLE
Create Shipper

### PR DESCRIPTION
**Pull request by Sapmetappels**
Gets the customer and order number from the receiver. Also prints the numbers for convenience. Also includes getter.

### CODE REVIEW

**Comment by Sapmetappels** [comment]
First version Shipper.

File: Shipper [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package physicalInternet;
+
+public class Shipper {
+	public static void main(String[] args) {
```
Comment (N-Wouda) [inline Shipper]: I'd put the main in a new file altogether, and _not_ in any of these classes.

File: Shipper [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		System.out.println(customerNumber);
+		System.out.println(orderNumber);
+	}
+
+	private static int customerNumber = Receiver.getCustomerNumber();
```
Comment (N-Wouda) [inline Shipper]: More generally, do not use `static` here. These are properties of the objects, not of the classes themselves.

**Comment by Sapmetappels** [comment]
Fixed the comments of Niels.

**Comment by N-Wouda** [comment]
Could you include your main file in any one of these pull requests? (generally just open one pull request for all this, it makes reviewing tasks much easier)

**Comment by N-Wouda** [comment]
I'd suggest the following file structure:

 ```java
main.java
/hub
    hub.java
/receiver
    receiver.java
/shipper
    shipper.java 
```
Where `/name` is a folder.

**Pull request closed without merging**